src/parameterized_tendencies/radiation/radiation.jl

```diff
@@ -185,19 +185,19 @@ function radiation_model_cache(
         end
 
         if idealized_insolation # perpetual equinox with no diurnal cycle
-            solar_zenith_angle = FT(π) / 3
+            cos_zenith = cos(FT(π) / 3)
             weighted_irradiance =
                 @. 1360 * (1 + FT(1.2) / 4 * (1 - 3 * sind(latitude)^2)) /
-                   (4 * cos(solar_zenith_angle))
+                   (4 * cos_zenith)
         else
-            solar_zenith_angle = weighted_irradiance = NaN # initialized in callback
+            cos_zenith = weighted_irradiance = NaN # initialized in callback
         end
 
         radiation_model = RRTMGPI.RRTMGPModel(
             rrtmgp_params,
             data_loader,
             # TODO: pass FT through so that it can be controlled at the top-level
-            Float64,
+            FT,
```

Comment:
Let's maybe apply this change in a separate PR, since this will impact our Float32 jobs